apps/settings/js/panels/full_developer_mode/panel.js

```diff
@@ -0,0 +1,45 @@
+define(function(require) {
+  'use strict';
+
+  var SettingsService = require('modules/settings_service');
+  var SettingsCache = require('modules/settings_cache');
+  var DialogService = require('modules/dialog_service');
+  var Panel = require('modules/panel');
+
+  return function() {
+    return Panel({
+      onInit: function(panel) {
+        this._panel = panel;
+        SettingsCache.getSettings((results) => {
+          var devMenuEnabled = !!results['developer.menu.enabled'];
+          if (devMenuEnabled) {
+            this._showFullDevModeWarning();
+          } else {
+            this._showEnableDevMenuWarning();
+          }
+        });
+      },
+      _close: function() {
+        SettingsService.navigate('home');
```

Comment:
Because this panel is opened as webActivity, why we have to navigate to `home` page after user clicking on cancel button ? Shouldn't we directly dismiss this webActivity and user will be redirected to original app ?